tasks/jasmine.js

```diff
@@ -194,7 +195,7 @@ module.exports = function(grunt) {
 
     phantomjs.on('console', function(msg) {
       thisRun.cleanConsole = false;
-      if (options.display === 'full') {
+      if (options.showLogging) {
```

Comment:
Ok great! I thought you were against the use of `options.display` to determine whether logs should show, but I do like this specific check, in order to preserve the current behavior!